core/domain/services/graphql/connections/RootQueryFormSectionsConnection.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace EventEspresso\core\domain\services\graphql\connections;
+
+use EEM_Form_Section;
+use EventEspresso\core\domain\services\graphql\connection_resolvers\FormSectionConnectionResolver;
+use EventEspresso\core\domain\services\graphql\abstracts\AbstractRootQueryConnection;
+use Exception;
+
+/**
+ * Class RootQueryFormSectionsConnection
+ * Description
+ *
+ * @package EventEspresso\core\domain\services\graphql\connections
+ * @author  Manzoor Wani
+ * @since   $VID:$
+ */
+class RootQueryFormSectionsConnection extends AbstractRootQueryConnection
+{
+
+
+    /**
+     * FormSectionConnection constructor.
+     *
+     * @param EEM_Form_Section               $model
+     */
+    public function __construct(EEM_Form_Section $model)
+    {
+        $this->model = $model;
+    }
+
+
+    /**
+     * @return array
+     * @since $VID:$
```

Comment:
can remove these on methods if the entire class is new and has an `@since` tag.

this is only needed on methods when a new method is added to an existing class

Reply:
OK, thanks